exercises/atbash-cipher/.meta/generator/atbash_cipher_case.rb

```diff
@@ -0,0 +1,27 @@
+require 'generator/exercise_case'
+
+class AtbashCipherCase < Generator::ExerciseCase
+  def workload
+    case property
+    when 'encode' then encode_workload
+    when 'decode' then decode_workload
+    else raise 'unexpected property encountered'
+    end
+  end
+
+  def encode_workload
+    [
+      "plaintext = '#{phrase}'",
+      "ciphertext = '#{expected}'",
+      "assert_equal ciphertext, AtbashCipher.encode(plaintext)"
+    ].join("\n")
+  end
+
+  def decode_workload
+    [
+      "ciphertext = '#{phrase}'",
+      "plaintext = '#{expected}'",
```

Comment:
Changing the order on these variables for the two sections keeps it in line with the structure from the `affine-cipher` tests.  I think it also is nice for the student to see the variables appear in an order consistent with the variables' relationships to the `assert_equal` call.

Reply:
this is fine.